libs/remix-ui/vertical-icons-panel/src/lib/components/RequiredSection.tsx

```diff
@@ -0,0 +1,67 @@
+import { VerticalIcons } from 'libs/remix-ui/vertical-icons-panel/types/vertical-icons-panel'
+// eslint-disable-next-line no-use-before-define
+import React, { Fragment, MutableRefObject } from 'react'
+import { Chevron } from './Chevron'
+import Debugger from './Debugger'
+import FilePanel from './FilePanel'
+import PluginManager from './PluginManager'
+import Solidity from './Solidity'
+import SolidityStaticAnalysis from './SolidityStaticAnalysis'
+import Udapp from './Udapp'
+
+interface RequiredSectionProps {
+  verticalIconsPlugin: VerticalIcons
+  itemContextAction: (e: any, name: string, documentation: string) => Promise<void>
+  addActive: (name: string) => void
+  removeActive: () => void
+  scrollableRef: MutableRefObject<any>
+}
+
+function RequiredSection ({ verticalIconsPlugin, itemContextAction, addActive, removeActive, scrollableRef }: RequiredSectionProps) {
+  return (
+    <Fragment>
+      <FilePanel
```

Comment:
As an improvement, we can refactor the required section to reuse a single Icon component with changing props.